frontend/src/metabase/visualizations/components/legend/LegendItemDot.styled.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from "@emotion/styled";
+
+export const OuterCircle = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 12px;
+  height: 12px;
+  border-radius: 50%;
+  background-color: var(--mb-color-border);
```

Comment:
I think we could possibly make this a little darker. Behaviorally it feels great but once the color is off it can be a little hard to find on a squint test.

Reply:
@kdoh just pushed the change, what do you think?

https://github.com/user-attachments/assets/52c2f950-dbd4-4374-a439-317a7368d1dd